themes/axin/README.md

```diff
@@ -0,0 +1,5 @@
+# Axi-Dark
+
+![alt text](axin-dark.png)
+
+### _Axin Bash Prompt, inspired by theme "Sexy" and "Bobby"_
```

Comment:
The headings shouldn't be used as an emphasis. That breaks the document structure. Also, I don't think we need to emphasize this.